docs/Model_Assumptions.rst

```diff
@@ -139,6 +138,34 @@ Also, if you are aiming at very high minimal renewable factors, the simulation t
 
 The minimum renewable share is introduced to the energy system by `D2.constraint_minimal_renewable_share()` and a validation test is performed with `E4.minimal_renewable_share_test()`.
 
+Minimal degree of autonomcy constraint
+######################################
+
+The minimal degree of autonomy constraint requires the capacity and dispatch optimization of the MVS to reach at least the minimal degree of autonomy defined within the constraint. The degree of autonomy of the optimized energy system may also be higher than the minimal degree of autonomy. Please find the definition of here: :ref:`_kpi_degree_of_autonomy`
+
+The minimal degree of autonomy is applied to the whole, sector-coupled energy system, but not to specific sectors. As such, energy carrier weighting plays a role and may lead to unexpected results. The constraint reads as follows:
+
+.. math::
+        minimal degree of autonomy <= \frac{\sum local demand \cdot weighting factor - \sum consumtion from energy providers \cdot weighting factor}{\sum local demand  \cdot weighting factor}
```

Comment:
The degree of autonomy of the optimized energy system may also be higher than the minimal degree of autonomy.

So shouldn't it be : minimal degree of autonomy => ...

Reply:
But that would mean that the real system autonomy is lower than the minimal degree of autonomy. This one is correct.

min DOA <= DA of system, potentially higher

Reply:
Extended equation:

```
minimal degree of autonomy <= degree of autonomy of system = \frac{\sum local demand \cdot weighting factor - \sum consumtion from energy providers \cdot weighting factor}{\sum local demand  \cdot weighting factor}
```